dpdata/system.py

```diff
@@ -806,3 +858,11 @@ def check_LabeledSystem(data):
        assert( len(data['cells']) == len(data['coords']) == len(data['energies']) )
 
 
+def elements_index_map(elements,standard=False,inverse=False):
```

Comment:
please add unit test for this function. The following cases are not covered by the current unit tests:
1, standard == False, inverse == True
2, standard == False, inverse == False 
3, standard == True, inverse = True